lumigator/frontend/src/services/models/modelsService.ts

```diff
@@ -8,7 +8,8 @@ async function fetchModels(task_name = 'summarization') {
       return response.data.items;
     }
   } catch (error) {
-    throw new Error('Fetching Models failed.', { cause: error });
+    console.error('Fetching Models failed.', { cause: error });
+    throw error;
```

Comment:
`new Error` expects only one param which is a string, so I just logged the error and bubbled it to the caller